Source/svg/SVGParser.swift

```diff
@@ -262,34 +262,34 @@ open class SVGParser {
                 if let rule = getFillRule(styleAttributes) {
                     path = Path(segments: path.segments, fillRule: rule)
                 }
-                return Shape(form: path, fill: getFillColor(styleAttributes, groupStyle: styleAttributes), stroke: getStroke(styleAttributes, groupStyle: styleAttributes), place: position, opacity: getOpacity(styleAttributes), clip: getClipPath(styleAttributes), tag: getTag(element))
+                return Shape(form: path, fill: getFillColor(styleAttributes, groupStyle: styleAttributes), stroke: getStroke(styleAttributes, groupStyle: styleAttributes), place: position, opacity: getOpacity(styleAttributes), clip: getClipPath(styleAttributes, boundingBox: path.bounds()), tag: getTag(element))
```

Comment:
You actually slow down parser with such an approach, because now you calculate locus bounds every time you create a `Shape` and it might be time consuming. Instead pass `Locus` to a `getClipPath()` method and call `.bounds()` only when it necessary.